### PR TITLE
QA Fail/Problems on switching users

### PR DESCRIPTION
**Pull request by PhotoNomad0**
## Describe what your pull request addresses

- [ ] Fix to only show workspace when user is logged in and we have user data
- [ ] fixes to useUserLocalStorage to initialize with saved value from local Storage
- [ ] updated single-scripture-rcl with scripture history fixes

## Test Instructions
- can test with https://deploy-preview-178--gateway-edit.netlify.app/
- [ ] follow Kos's QA fail steps in https://github.com/unfoldingword/gateway-edit/issues/98
- [ ] do several refreshes of screen to make sure saved settings reload correctly each time

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/gateway-edit/178)

### CODE REVIEW

File: src/components/WorkspaceContainer.js [hunk 1]
```diff
@@ -170,175 +177,178 @@ function WorkspaceContainer() {
...
-          resource={{
-            owner,
-            languageId,
-            resourceId: TARGET_LITERAL,
-            originalLanguageOwner: scriptureOwner,
-          }}
-          {...commonScriptureCardConfigs}
-        />
+        {loading || content || error ?
+          <DraggableCard
```
Comment (mannycolon) [inline src/components/WorkspaceContainer.js:191]: Why is the draggable card use for?

**Review by mannycolon — APPROVED** [review]

File: src/components/WorkspaceContainer.js (hunk 1 above)
Comment (PhotoNomad0) [reply src/components/WorkspaceContainer.js:191]: @mannycolon - github compare is a mess. you can see it on line 180 of the previous version.  Something you added 3/29.

File: src/components/WorkspaceContainer.js [hunk 2]
```diff
@@ -134,8 +138,11 @@ function WorkspaceContainer() {
       } else {
         console.log(`no bibles`)
       }
-    })// eslint-disable-next-line
-  }, [owner, languageId, branch, server])
+      setWorkspaceReady(true)
+    }).catch(() => {
```
[the remark this answers is not in the document]
Comment (PhotoNomad0) [reply src/components/WorkspaceContainer.js:142]: No, the intent here was to catch any request errors and recover gracefully by showing the workspace.

**Review by mannycolon — APPROVED** [review]
:lgtm:

> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/gateway-edit/178#-:-MZTI3k48SHLZw4R7YAV:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @PhotoNomad0)

___
*[src/components/WorkspaceContainer.js, line 191 at r1](https://reviewable.io/reviews/unfoldingword/gateway-edit/178#-MZTGxKPf8vXtejV-LYs-r1-191:-MZTHilJ3Y6xbf5G1ehX:bg0xcwg) ([raw file](https://github.com/unfoldingword/gateway-edit/blob/ff9ad2de21e476d18d7128766c53e91034b1391b/src/components/WorkspaceContainer.js#L191)):*
<details><summary><i>Previously, PhotoNomad0 (Bruce McLean) wrote…</i></summary><blockquote>

@mannycolon - github compare is a mess. you can see it on line 180 of the previous version.  Something you added 3/29.
</blockquote></details>

Oh okay, that is the draggable card used for the article links, totally forgot about this.

**Comment by mannycolon** [comment]
Found a problem when logging in with a user and then logging out and logging in with a different user cause the user to see the resource workspace first and then quickly changing to account setting to select organization and language. The user should not see the resource workspace before selecting the account settings. 

Steps to reproduce:

1. Clear your local storage
2. Login with user1 
3. Log out user1
4. Login with user2


See attached video: 
https://user-images.githubusercontent.com/17072300/116581088-f9fac000-a8e1-11eb-9b40-cc1b66504236.mp4

**Pull request merged**